.travis/install-aom.sh

```diff
@@ -1,27 +1,21 @@
 #!/bin/bash
 set -ex
 
-AOM_VERSION="1.0.0.errata1-3~18.04.york0"
-PKG_URL="http://ppa.launchpad.net/jonathonf/ffmpeg-4/ubuntu/pool/main/a/aom"
-
-case "$ARCH" in
-  x86_64) ARCH=amd64 ;;
```

Comment:
Why are we removing x86_64 case, as it is in GitHub Actions?

Reply:
Exactly, we are testing it with `GitHub Actions`